package/install_osx.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -ev
+brew update
+brew install python3
+sudo pip3 install --upgrade pip setuptools
```

Comment:
Hi @carlosperate, quick question -- what's this for, and why does it need sudo?  (Okay, two quick questions!)